### PR TITLE
support metrics tooltip

### PR DESCRIPTION
**Pull request by zyguan**
Signed-off-by: zyguan <zhongyangguan@gmail.com>

Also move the template to this repo for better maintainability.

### CODE REVIEW

**Comment by zyguan** [comment]
@cfzjywxk @you06 PTAL

**Comment by cfzjywxk** [comment]
That's quite useful. 
@you06 PTAL

File: template.html [hunk 1]
```diff
@@ -0,0 +1,63 @@
...
+<div id="CONTENT"></div>
+<script id="MARKDOWN" type="text/plain">
+__MARKDOWN__
+</script>
+
+<script>
+  // export railroad defaults
+  Object.entries(railroad.default).forEach(([k, v]) => globalThis[k] = v);
+  // aliases
+  function Span(text, {href, tooltip, color}={}) {
```
Comment (you06) [inline template.html:33]: Need to replace all `NonTerminal` with `Span` in the future?

Comment (zyguan) [reply template.html:33]: I've already replaced them. Here I mean to define shortcuts for some common usages, I think we can still use the builtin diagram node (like `NonTerminal`) if needed.

**Review by you06 — APPROVED** [review]

**Pull request merged**